api/app/Models/PoolCandidate.php

```diff
@@ -270,6 +270,20 @@ public function filterByEquity(Builder $query, array $equity): Builder
         return $query;
     }
 
+    public function scopePoolCandidateStatus(Builder $query, ?string $poolCandidateStatus): Builder
+    {
+        if(empty($poolCandidateStatus)){
+            return $query;
+        }
+
+        if($poolCandidateStatus == 'AVAILABLE'){
+            $query->where('pool_candidate_status', 'AVAILABLE');
+            return $query;
+        }
+
+        return $query;
+    }
+
```

Comment:
Take this with a grain of salt, as maybe I'm missing some consideration, but: Since this only handles "available" status, why not just make a `scopeAvailable` with no args?

```suggestion
    public function scopeAvailable(Builder $query): Builder
    {
        $query->where('pool_candidate_status', 'AVAILABLE');
    }

```

Reply:
Realizing I don't understand something about graphql setup that makes this required, but also would suggest it deal with all types so as not to be misleading